lib/features/login/presentation/screens/login_screen.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:events_finder/features/home/presentation/cubit/home_cubit.dart';
+import 'package:events_finder/features/home/presentation/screens/home_screen.dart';
+import 'package:events_finder/features/login/presentation/cubit/login_cubit.dart';
+import 'package:events_finder/shared/di/service_locator.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LoginScreen extends StatefulWidget {
+  const LoginScreen({Key? key}) : super(key: key);
+
+  @override
+  State<LoginScreen> createState() => _LoginScreenState();
+}
+
+class _LoginScreenState extends State<LoginScreen> {
+  _login() => context.read<LoginCubit>().login();
+
+  _onUsernameChanged(String username) =>
+      context.read<LoginCubit>().onUsernameChanged(username);
+
+  _onPasswordChanged(String password) =>
+      context.read<LoginCubit>().onPasswordChanged(password);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocListener<LoginCubit, LoginState>(
+      listener: (context, state) {
+        if (state is LoginSuccess) {
+          Navigator.pop(context);
```

Comment:
why you pop here?

Reply:
When login succeeded, I pop the login screen to remove it from the widget stack to avoid going back to login  on physical back pressed